src/components/GlobalData/index.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import axios from "axios";
+import ETHGasPrice from "../ETHGasPrice";
+import { CurrencySelect } from "../CurrencySelect";
+import { CaretSymbol } from "../CaretSymbol";
+import formatNumber from "../../uitls/NumberUtils/formatNumber";
+import convertLongNumber from "../../uitls/NumberUtils/convertLongNumber";
+import getCurrencySymbol from "../../uitls/getCurrencySymbol";
+import { Row, Col, Tooltip } from "antd";
+import {
+  GlobalDataBar,
+  ETHGasPriceCol,
+  Container,
+  Description,
+  Value,
+  Ticker,
+  MarketCapChange,
+} from "./GlobalData.styles";
+
+export default class GlobalData extends React.Component {
+  state = {
+    globalData: null,
```

Comment:
Same thing here, doesnt it make more sense to initialise all the properties in the initial state object?